### PR TITLE
Changes to Airbyte module related to the taxonomy changes.

### PR DESCRIPTION
**Pull request by revit13**
/Closes https://github.com/fybrik/fybrik/issues/1866
Relates to [#1940](https://github.com/fybrik/fybrik/issues/1940#event-8341658499)

This PR contains changes to Airbyte module related to the taxonomy changes:
The connection types that are supported by the module in [module.yaml](https://github.com/fybrik/airbyte-module/blob/main/module.yaml) should now be appropriately defined in pkg/storage/layers/connection.yaml file as an additional layer to Fybrik base taxonomy.
Custom properties for the connection types can be defined in values.yaml file of the module helm chart and would be merged with the properties defined in the Fybrik taxonomy connection.

### CODE REVIEW

File: helm/abm/values.sample.read.mysql.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+- args:
+  - connection:
+        name: mysql
+        mysql:
+          database: test
+          host: host.docker.internal
+          port: 3306
+  assetID: new-data-parquet
+  capability: read
+
```
Comment (shlomitk1) [inline helm/abm/values.sample.read.mysql.yaml:16]: reference to credentials?

File: helm/abm/templates/_helpers.tpl [hunk 2]
```diff
@@ -30,3 +30,25 @@ Create chart name and version as used by the chart label.
...
+additional properties defined for a connector. If the additional properties does not
+contain a format key then they are considered equal.
+*/}}
+{{- define "airbyte-module.is-format-equal" -}}
+{{- if hasKey .additionalProperties "expected_format" -}}
+{{- if eq (get .additionalProperties "expected_format") .connectionFormat -}}
+{{- true }}
+{{- end -}}
+{{- else -}}
+{{- true }}
```
Comment (shlomitk1) [inline helm/abm/templates/_helpers.tpl:45]: true in both cases?

File: helm/abm/files/conf.yaml [hunk 3]
```diff
@@ -1,14 +1,59 @@
...
+         {{- end }}
+         {{- toYaml .connection | nindent 6 }}
+       {{- end }}
     {{- end }}
-    {{- end }}
-{{- end -}}
+{{- end }}
 {{- else -}}
 data: []
 {{- end -}}
```
Comment (Mohammad-nassar10) [inline helm/abm/files/conf.yaml:59]: Is it better to do the parsing here in `conf.yaml` or to pass all the values to the module and do the parsing in the module's code (as it was before the changes)?

Comment (revit13) [reply helm/abm/files/conf.yaml:59]: I think @cdoron preferred that the module code will not be affected by the connector type this is why all the translation is done in the chart.

File: helm/abm/values.sample.read.mysql.yaml (hunk 1 above)
Comment (revit13) [reply helm/abm/values.sample.read.mysql.yaml:16]: I'll add it.

File: helm/abm/templates/_helpers.tpl (hunk 2 above)
Comment (revit13) [reply helm/abm/templates/_helpers.tpl:45]: Yes, if the key is not present or its present and equal.

File: helm/abm/values.sample.read.mysql.yaml [hunk 4]
```diff
@@ -0,0 +1,24 @@
+# Try it with `helm install --generate-name --dry-run -f helm/abm/values.sample.yaml helm/abm`
```
Comment (cdoron) [inline helm/abm/values.sample.read.mysql.yaml:1]: Maybe:
```
# Try it with `helm install --generate-name --dry-run -f helm/abm/values.sample.read.mysql.yaml helm/abm`
```

File: helm/abm/values.sample.write.mysql.yaml [hunk 5]
```diff
@@ -0,0 +1,24 @@
+# Try it with `helm install --generate-name --dry-run -f helm/abm/values.sample.yaml helm/abm`
```
Comment (cdoron) [inline helm/abm/values.sample.write.mysql.yaml:1]: Maybe:
```
# Try it with `helm install --generate-name --dry-run -f helm/abm/values.sample.write.mysql.yaml helm/abm`
```

File: helm/abm/values.sample.read.mysql.yaml (hunk 4 above)
Comment (revit13) [reply helm/abm/values.sample.read.mysql.yaml]: Thanks fixed

File: helm/abm/values.sample.write.mysql.yaml (hunk 5 above)
Comment (revit13) [reply helm/abm/values.sample.write.mysql.yaml]: Thanks fixed

**Review by cdoron — APPROVED** [review]

File: fybrik/README.md [hunk 6]
```diff
@@ -15,12 +15,11 @@ You will need a copy of the Fybrik repository (`git clone https://github.com/fyb
 
 1. Install Fybrik Prerequisites. Follow the instruction in the Fybrik [Quick Start Guide](https://fybrik.io/dev/get-started/quickstart/). Stop before the "Install control plane" section.
 
-1. Before installing the control plane, we need to customize the [Fybrik taxonomy](https://fybrik.io/dev/tasks/custom-taxonomy/), to define new connection and interface types. This customization requires [go](https://go.dev/dl/) version 1.17 or above. Please note that in this tutorial a built-in catalog called `katalog` is used as opposed to the default Openmetadata catalog used in Fybrik. Run:
+1. Install Fybrik with built-in catalog called `katalog`. This is used as opposed to the default Openmetadata catalog used in Fybrik.
```
Comment (shlomitk1) [inline fybrik/README.md:18]: Install Fybrik with a built-in catalog, called Katalog, as opposed to the Openmetadata catalog which is installed by default.

Comment (revit13) [reply fybrik/README.md]: Fixed. Thanks

File: helm/abm/files/conf.yaml (hunk 3 above)
Comment (Mohammad-nassar10) [reply helm/abm/files/conf.yaml:53]: Can you add an example file of how this conf.yaml file might look? (just a suggestion, if it needs much work you can ignore it).

Comment (revit13) [reply helm/abm/files/conf.yaml:53]: @Mohammad-nassar10  `sample/read_mysql.yaml` is an example of a generated conf file.  ``helm install --generate-name --dry-run -f helm/abm/values.sample.read.mysql.yaml helm/abm` can be used to generate this sample. Please let me know if you think I should add more info. Thanks

**Review by Mohammad-nassar10 — APPROVED** [review]

**Review by shlomitk1 — APPROVED** [review]

**Pull request merged**